recipe/symfony.php

```diff
@@ -22,17 +26,29 @@
 
 // Assets
 set('assets', ['web/css', 'web/images', 'web/js']);
-// Default true - BC for Symfony < 3.0
-set('dump_assets', true);
+
+// Requires non symfony-core package `kriswallsmith/assetic` to be installed
+set('dump_assets', false);
```

Comment:
I mentioned in the PR summary that we shouldn't assume asset compilation is desired b/c it isn't a core Symfony command and there is no reason to believe the user has it installed (in any Symfony version). 

That command relies on https://github.com/kriswallsmith/assetic and https://github.com/symfony/assetic-bundle, which may not (and very often isn't) installed.

Behavior is already changing with this pull request, as before the `deploy:assetic:dump` command was *never* called on Symfony >= 2.8. Forcing it by default would be the same as calling `database:migrate` by default, which we don't do (for good reason).

Previous behavior was:
- For all versions: `dump_assets` is true
- For Symfony < `2.8`: via `deploy:assetic:dump`, call `app/console assetic:dump`
- For Symfony >= `2.8`: via an overridden `deploy:assetic:dump, call `app/console assets:install`

The above is inconsistent, odd, and incorrect behavior.

The new behavior is:
- For all versions `dump_assets` is false
- For all versions, via `deploy:assets:install`, call `app/console assets:insall` (core command)
- For all versions, via `deploy:assetic:dump`, call `app/console assetic:dump` IF the user enables it via the above environment var (non-core command)